### PR TITLE
Non suppressed register

### PR DESCRIPTION
**Pull request by slubwama**
This is a recreated because of an accidental merge but later reverted the Previous #340

Ticket https://trello.com/c/Bo3R6bba/661-add-changes-of-the-revised-viral-load-non-suppressed-register

### CODE REVIEW

File: api/src/main/resources/metadata/Custom_Concepts.xml [hunk 1]
```diff
@@ -322,6 +322,42 @@
...
+    <concept  concept_id="164989"  retired="0"  datatype_id="2"  class_id="5"  is_set="0"  creator="1"  date_created="2017-11-23 12:46:38"  version="1.0"  changed_by="1"  date_changed="2017-11-23 12:46:38" uuid="164989AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+    <concept  concept_id="164988"  retired="0"  datatype_id="2"  class_id="5"  is_set="0"  creator="1"  date_created="2017-11-23 12:45:43"  version="1.0"  changed_by="1"  date_changed="2017-11-23 12:45:43" uuid="164988AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+    <concept  concept_id="164987"  retired="0"  datatype_id="2"  class_id="5"  is_set="0"  creator="1"  date_created="2017-11-23 12:44:15"  version="1.0"  changed_by="1"  date_changed="2017-11-23 12:44:15" uuid="164987AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+    <concept  concept_id="164986"  retired="0"  datatype_id="3"  class_id="5"  is_set="0"  creator="1"  date_created="2017-11-23 12:42:49"  version="1.0"  changed_by="1"  date_changed="2017-11-23 12:44:05" uuid="164986AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+
+    <!--KP-->
```
Comment (ssmusoke) [inline api/src/main/resources/metadata/Custom_Concepts.xml]: @slubwama KP section is duplicated here

File: api/src/main/resources/metadata/Custom_Concepts.xml [hunk 2]
```diff
@@ -3503,6 +3610,48 @@
...
+    <concept_answer  concept_answer_id="6860"  concept_id="164989"  answer_concept="1066" creator="1"  date_created="2017-11-23 12:46:38"  sort_weight="2"  uuid="6860CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC"/>
+    <concept_answer  concept_answer_id="6857"  concept_id="164988"  answer_concept="1065" creator="1"  date_created="2017-11-23 12:45:43"  sort_weight="1"  uuid="6857CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC"/>
+    <concept_answer  concept_answer_id="6858"  concept_id="164988"  answer_concept="1066" creator="1"  date_created="2017-11-23 12:45:43"  sort_weight="2"  uuid="6858CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC"/>
+    <concept_answer  concept_answer_id="6855"  concept_id="164987"  answer_concept="1066" creator="1"  date_created="2017-11-23 12:44:15"  sort_weight="2"  uuid="6855CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC"/>
+    <concept_answer  concept_answer_id="6856"  concept_id="164987"  answer_concept="1065" creator="1"  date_created="2017-11-23 12:44:15"  sort_weight="1"  uuid="6856CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC"/>
+
+    <!--KP-->
```
Comment (ssmusoke) [inline api/src/main/resources/metadata/Custom_Concepts.xml]: @slubwama KP section duplicated here

**Comment by ssmusoke** [comment]
@slubwama 

1. Change Unique Identifier to National ID and the Javascript validation rules on the registration page 

2. VL Results at Initiation can be less than 1000 however must be more than 20 

3. Date validations should be based on the Registration date (encounter date) since its the date that does not change. Show the messages on those dates rather than the Encounter date 

4. Restart IAC is a required field too 

5. IAC Session - validate the session date, assessment score, and adherence scores for the fields that are shown 

6. VL Results after IAC: (Merge the column to separate the two)  Add labels for result and copies 

7. Other Clinic Decision and Other Outcome - put the labels at the same level as the fields

**Comment by slubwama** [comment]
Too much validation on this form will make it unusable.We have to strike a balance.

**Comment by ssmusoke** [comment]
@slubwama Then provide suggestions, remembering that if the data is not valid, then you cannot use it for reporting or analysis

**Comment by slubwama** [comment]
@ssmusoke  6. and 7 don't add any value. So not going to give them much effort

**Comment by slubwama** [comment]
@ssmusoke 2. is never clear. It depends on a machine that has done VL other than the numbers. Its better to leave it open

**Pull request closed without merging**